imsim/atmPSF.py

```diff
@@ -196,7 +216,9 @@ def __eq__(self, rhs):
 
     @staticmethod
     def _vkSeeing(r0_500, wavelength, L0):
-        # von Karman profile FWHM from Tokovinin fitting formula
+        # von Karman profile FWHM from fitting formula in eqn 19 of
+        # Tokovinin 2002, PASP, v114, p1156
+        # https://dx.doi.org/10.1086/342683
         kolm_seeing = galsim.Kolmogorov(r0_500=r0_500, lam=wavelength).fwhm
         r0 = r0_500 * (wavelength/500)**1.2
```

Comment:
Is this right?  Not (`1+self.seeing`)?  Also above in `_build_atm` you use 6/5.

Reply:
I'm lost, where do you suggest I add `1+self.seeing`?  Happy to change 6/5 -> 1.2.

This bit is implementing the following:
<img width="176" alt="Screen Shot 2023-07-20 at 11 57 03 AM" src="https://github.com/LSSTDESC/imSim/assets/3650485/dc2fabcc-f01c-4054-a5f8-bcece81caf48">

where \epsilon_0 is the fwhm from a Kolmogorov profile and \epsilon_vK is with the outer scale added in (a VonKarman profile).

Reply:
OK.  I guess I just assumed the 1.2 was related to the 0.2 of Kolmogorov seeing.  If not, ignore me.

Reply:
It's just the normal 6/5.  I guess 6/5 might be a bit clearer.